org.eclipse.scout.rt.uinotification/src/main/java/org/eclipse/scout/rt/api/uinotification/UiNotificationRegistry.java

```diff
@@ -185,14 +188,14 @@ protected List<UiNotificationDo> get(String topic, String user, final List<UiNot
   /**
    * Creates a notification to mark the start of the subscription.
    * <p>
-   * This is necessary to ensure the client receives every notification from now on even if the connection
-   * temporarily drops before the first real notification can be sent.
-   * During that connection drop a notification could be added that needs to be sent as soon as the connection is reestablished again.
+   * This is necessary to ensure the client receives every notification from now on even if the connection temporarily
+   * drops before the first real notification can be sent. During that connection drop a notification could be added
+   * that needs to be sent as soon as the connection is reestablished again.
    * <p>
-   * The result can contain multiple notifications, one for each cluster node.
-   * So, if a cluster node created notifications for the topic, the last one for that node will be returned.
```

Comment:
Why have you adjusted this? I find it more readable, if every sentence is on one line. We do the same for markdown and ascii doc text.